charts/cloudnative-pg/templates/rbac.yaml

```diff
@@ -378,4 +378,54 @@ subjects:
 - kind: ServiceAccount
   name: {{ include "cloudnative-pg.serviceAccountName" . }}
   namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I'd guard the creation of these ClusterRoles with a variable, either using `rbac.aggregateClusterRoles` or adding a second flag, defaulting to `false`. So that people upgrading to the new version won't grant additional permissions to their users by mistake.

Reply:
(Cluster)Roles do not grant any permissions without a matching (Cluster)RoleBinding. Maybe defining `rbac.aggregateClusterRoles` to false by default would be enough to achieve your requirement without needing an extra values variable?

Reply:
Sounds good to me, if you can change that, @NaPs :+1:  I'd say this could also be backported to the plain manifest installation method, I'll open an issue for that.